### PR TITLE
Updates dependencies: Material UI lab and charts.js

### PR DESCRIPTION
**Pull request by mhhutton**
[screenshot](file:///home/mhhutton/Pictures/componentSS.png)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,15 @@
...
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.58",
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
+    "chart.js": "^3.3.2",
     "express": "^4.17.1",
     "jquery": "^3.1.1",
     "prop-types": "^15.7.2",
     "react": "^16.4",
+    "react-chartjs-2": "^3.0.3",
```
Comment (Pix3lAssassin) [inline package.json:41]: Do we need both charts?

File: client/src/components/reviews/RatingsAndReviews.jsx [hunk 2]
```diff
@@ -1,182 +1,199 @@
...
+
+  //       {/* End of Right Panel */}
+  //       </div>
+
+  //     {/* End of midContainer */}
+  //     </div>
+
+  //   {/* End of Parent Container */}
+  //   </div>
+  // );
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/RatingsAndReviews.jsx:197]: Are you still using this?

File: client/src/components/reviews/StarBarChart.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    {
+      label: '',
+      data: [20, 40, 10, 25, 5],
+      backgroundColor: [
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/StarBarChart.jsx:29]: Might be a good idea to create a variable for this

File: package-lock.json [hunk 4]
```diff
@@ -1,14559 +1,8 @@
 {
   "name": "front-end-capstone",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
```
Comment (Pix3lAssassin) [inline package-lock.json:5]: I don't think this file should be committed or it needs to be updated

File: client/src/components/reviews/RatingsAndReviews.jsx [hunk 5]
```diff
@@ -1,182 +1,199 @@
...
-    alignContent:"center"
-  }
-}));
+//Component imports
+import ReviewTile from './ReviewTile.jsx';
+import TotalReviewCount from './TotalReviewCount.jsx';
+import ReviewPercentageAndStars from './ReviewPercentageAndStars.jsx';
+import StarBarChart from './StarBarChart.jsx';
+import ProductReviewChart from './ProductReviewChart.jsx';
+//import '../../../dist/styles.css';
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/RatingsAndReviews.jsx:13]: Like the organization

File: client/src/components/reviews/ProductReviewChart.jsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+const data = {
+  labels: ['Comfort:', 'Size'],
+  datasets: [
+    {
+      label: '',
+      data: [40, 40],
+      backgroundColor: [
+        'rgba(35, 203, 167, 1)',
+        'rgba(35, 203, 167, 1)',
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/ProductReviewChart.jsx:25]: Might consider using a variable for this

File: client/src/components/reviews/StarBarChart.jsx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+      display: false,
+    },
+    yAxis: {
+      display: true,
+      grid: {
+        display: false
+      }
+    }
+  }
+};
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/StarBarChart.jsx:72]: You are using the same code as in ProductReviewChart

File: client/src/components/reviews/ProductReviewChart.jsx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+      display: false,
+    },
+    yAxis: {
+      display: true,
+      grid: {
+        display: false
+      }
+    }
+  }
+};
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/ProductReviewChart.jsx:64]: You are using the same code as in StarBarChart

**Review by Pix3lAssassin — CHANGES_REQUESTED** [review]

**Comment by mhhutton** [comment]
Making changes. Will push with updates later.

**Pull request closed without merging**